app/assets/javascripts/config/angularProviders.js.coffee

```diff
@@ -13,7 +13,7 @@
         # This is set up to universally capture HTTP errors, particularly 503 or 504, when a bad request / timeout occurred.
         # It will pop up an alert and stop the spinning loader and re-enable short form inputs so that the user can try again.
         responseError: (error) ->
-          if error.status >= 500
+          if error.status >= 400
```

Comment:
@chrisdolendo why `400` error? I don't like the `error.status >= 400` because then for example a `404` will get an "uh oh!" which doesn't seem right. `4xx` error means "the client did something wrong", whereas `5xx` means "the server did something wrong" which is what seems to be the case here. So I think it should stay in the 500 range.

Reply:
this question has been resolved